docs/Admin.md

```diff
@@ -534,8 +534,32 @@ If you want to deeply customize the app header, the menu, or the notifications,
 
 React-admin offers predefined layouts for you to use:
 
+<figure>
```

Comment:
image maps aren't responsive, so using svg is the only way to make parts of an image clickable.